iib/web/static/api_v1.yaml

```diff
@@ -1143,6 +1143,9 @@ components:
           items:
             type: string
           example: ["v4.5-10-08-2021"]
+        fbc:
```

Comment:
The other properties are having `example` section. Should we add it here as well?

Reply:
Added example and default